modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -24,12 +24,17 @@ import {
 	useFormState,
 } from 'dynamic-data-mapping-form-renderer';
 import {EVENT_TYPES as CORE_EVENT_TYPES} from 'dynamic-data-mapping-form-renderer/js/core/actions/eventTypes.es';
-import moment from 'moment';
+import moment from 'moment/min/moment-with-locales';
 import React, {useMemo} from 'react';
 
-const convertInputValue = (fieldType, value) => {
+const convertInputValue = (fieldType, locale, value) => {
 	if (fieldType === 'date') {
-		const date = moment(value).toDate();
+		moment.locale(locale);
```

Comment:
Hey @rodrigopaulino, the frontend guys told me to avoid setting the locale this way, because it changes the moment's locale everywhere. You can do something like we do [this](https://github.com/liferay/liferay-portal/blob/542ac19ddfbb24263d429cb2041b65f6d8c42f73/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/util/date.es.js#L24
) instead

Reply:
Sure thing, @marcosapmf 
Thanks for the input.